cmd/utils.go

```diff
@@ -34,6 +35,8 @@ import (
 	"time"
 
 	humanize "github.com/dustin/go-humanize"
+	"github.com/gorilla/mux"
+	"github.com/minio/minio/cmd/logger"
```

Comment:
external imports and Minio internal imports should be blocked properly https://github.com/golang/go/wiki/CodeReviewComments#imports follow this style.